lcd/config_downloader.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import uuid
+import requests
+
+class ConfigDownloader:
+    def __init__(self, config_path, remote_url):
+        self.config_path = config_path
+        self.remote_url  = remote_url
+        self.api_key      = uuid.uuid4()
+
+    def run(self):
+        print "Setting up a new account"
+        print "please make sure an account with API key %s is configured" % self.api_key
```

Comment:
ok pefect, no was aware of that but didn't look close enough to see all steps were isolated.